lib/util/auth.js

```diff
@@ -5,19 +5,24 @@ import { isOtpMiddleware } from './user'
  * Custom links under the user account dropdown.
  * TODO: Determine/implement the urls below.
  */
-export const accountLinks = [
-  {
-    messageId: 'myAccount',
-    url: ACCOUNT_PATH
-  },
-  {
-    messageId: 'help',
-    // Add a target attribute if you need the link to open in a new window, etc.
-    // (supports the same values as <a target=... >).
-    // target: '_blank',
-    url: '/help'
-  }
-]
+export const accountLinks = (config) => {
+  const helpLink =
+    config?.extraMenuItems.find((link) => link.id === 'help') || undefined
+  const helpLinkUrl = helpLink ? helpLink.href : '/help'
+  return [
+    {
+      messageId: 'myAccount',
+      url: ACCOUNT_PATH
+    },
+    {
+      messageId: 'help',
+      // Add a target attribute if you need the link to open in a new window, etc.
+      // (supports the same values as <a target=... >).
+      // target: '_blank',
```

Comment:
Is this supposed to be uncommented?

Reply:
This exists in dev, but I think it's just a demonstration? Seems like weird behavior for this to open a new tab by default

Reply:
Good call. We can leave it commented out for now!